server/core/src/https/views/login.rs

```diff
@@ -617,7 +658,7 @@ async fn view_login_step(
                             let mut username_cookie =
                                 Cookie::new(COOKIE_USERNAME, session_context.username.clone());
                             username_cookie.set_secure(state.secure_cookies);
-                            username_cookie.set_same_site(SameSite::Strict);
+                            username_cookie.set_same_site(SameSite::Lax);
```

Comment:
I clearly missed this on a previous one, why aren't we using `Strict` here (and above, which is scarier)?

Reply:
https://developer.mozilla.org/en-US/docs/Web/Security/Practical_implementation_guides/Cookies#samesite

tl;dr is if it's strict then the cookie isn't sent on a redirect to kanidm. The redirector can't see the cookie, but lax allows it to be added on that redirection.

Reply:
Ah, 'k. With the other protections in place that'll work 😄 👍🏻